dremio_client/model/endpoints.py

```diff
@@ -269,9 +302,12 @@ def user(token, base_url, uid=None, name=None, ssl_verify=True):
     """
     if uid is None and name is None:
         raise TypeError("both id and name can't be None for a user call")
-    idpath = (uid if uid else "") + ", " + (".".join(name) if name else "")
-    endpoint = "/{}".format(uid) if uid else "/by-name/{}".format("/".join(name).replace('"', ""))
-    return _get(base_url + "/api/v3/user{}".format(endpoint), token, idpath, ssl_verify=ssl_verify)
+    if uid is not None:
```

Comment:
more pythonic to said `if uid:`

Reply:
in fact can probably be `endurl = base_url + "/api/v3/user/" + ("{}".format(uid) if uid else "by-name/{}".format(name)` and remove the if block entirely